README.md

```diff
@@ -29,7 +29,42 @@ On many systems, the following hugepages setup is sufficient:
 
 ### Option 1: Obtain Packages
 
-Debian packages for Gatekeeper aren't yet available, but will be soon.
+Debian packages for Gatekeeper aren't yet available, but will be soon at the
+[Releases](https://github.com/AltraMayor/gatekeeper/releases)
+page.
+
+#### Install
+
+Once the packages are downloaded, they can be installed with the commands below:
+
+    # dpkg -i libgkrte-*.deb \
```

Comment:
Using `sudo` here would emphasize that root is needed without saying anything else.

Reply:
Done. I also fixed the package versions to match what we're actually using.